_docs-as-code.rst

```diff
@@ -144,7 +155,8 @@ As an example, the ``doc/manifest.yaml`` file included in ``afw`` may look like
 .. code-block:: yaml
 
    # Name of the package and also name of the package doc directory
-   package: "afw"
+   # Note: N/A for `afw`.
+   # package: "afw"
```

Comment:
It feels a little strange to have a commented line in the doc that can never be uncommented.  Maybe don't included a note about a package section if one cannot exist?